docs/http/http-metrics.md

```diff
@@ -114,13 +120,24 @@ Tracing instrumentations that do so, MUST also set `http.request.method_original
 
 SHOULD NOT be set if only IP address is available and capturing name would require a reverse DNS lookup.
 
-**[5]:** Determined by using the first of the following that applies
+**[7]:** Determined by using the first of the following that applies
 
 - Port identifier of the [primary server host](/docs/http/http-spans.md#http-server-definitions) of the matched virtual host.
 - Port identifier of the [request target](https://www.rfc-editor.org/rfc/rfc9110.html#target.resource)
   if it's sent in absolute-form.
 - Port identifier of the `Host` header
 
+`http.request.error.reason` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
```

Comment:
Maybe add a caveat about limiting cardinality here.

Reply:
would be great, but this is auto-generated from `allow_custom_values: true` and we don't have a way to change it at the moment.

There is another note and brief requiring low cardinality for the values.